From b8034e235b057577130a73228dafdd5ca9e71b26 Mon Sep 17 00:00:00 2001
From: Raine Makelainen <raine.makelainen@jolla.com>
Date: Fri, 12 Jan 2018 10:51:35 +0200
Subject: [PATCH 2/2] Keep Qt5.6 requirement

There's no actual build requirement to update this to Qt5.7

Signed-off-by: Raine Makelainen <raine.makelainen@jolla.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2a3575d..86283f6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -18,7 +18,7 @@ include(KDECMakeSettings)
 
 include(GenerateExportHeader)
 
-set(REQUIRED_QT_VERSION 5.7.0)
+set(REQUIRED_QT_VERSION 5.6.0)
 find_package(Qt5Core ${REQUIRED_QT_VERSION} REQUIRED NO_MODULE)
 
 find_package(ZLIB)
-- 
2.7.4

